scripts/protocgen2.sh

```diff
@@ -1,11 +1,12 @@
-#!/usr/bin/env bash
+
 
 # this script is for generating protobuf files for the new google.golang.org/protobuf API
 
 set -eo pipefail
 
 protoc_gen_gopulsar() {
   go install github.com/cosmos/cosmos-proto/cmd/protoc-gen-go-pulsar@latest 2>/dev/null
+  go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest 2>/dev/null
```

Comment:
yes.

however, I think the ideal would be to use buf plugins (https://docs.buf.build/bsr/remote-generation/plugin-example), so that the docker image we call in `Makefile` is simply the latest buf docker image. then buf uses plugins to resolve each code generator (docker in docker would need to work i believe)

but i think this is fine for now

Reply:
100% we would need plugins for regen-proto and pulsar right?

Reply:
Yep